zerver/tests/test_subs.py

```diff
@@ -1655,6 +1655,25 @@ def test_subscriber_private_stream(self):
                                          invite_only=True)
         self.make_successful_subscriber_request(stream_name)
 
+    def test_json_get_subscribers(self):
+        # type: () -> None
+        """
+        json_get_subscribers also returns the list of subscribers for a stream.
+        """
+        stream_name = gather_subscriptions(self.user_profile)[0][0]['name']
+        expected_subscribers = gather_subscriptions(self.user_profile)[0][0]['subscribers']
+        result = self.client.post("/json/get_subscribers", {"stream": stream_name})
+        self.assert_json_success(result)
+        json = ujson.loads(result.content)
+        self.assertIn('subscribers', json)
+        self.assertIsInstance(json['subscribers'], list)
+        json_subscribers = [] # type: List[text_type]
+        for subscriber in json['subscribers']:
+            self.assertIsInstance(subscriber, six.string_types)
+            json_subscribers.append(subscriber)
+        self.assertEqual(len(json_subscribers) == len(expected_subscribers), True)
```

Comment:
No there is not any particular reason. 
But yes it would be a better option.

Thanks.